data/language/en-GB.txt

```diff
@@ -4168,6 +4168,9 @@ STR_5857    :{SMALLFONT}{BLACK}Game options
 STR_5858    :{SMALLFONT}{BLACK}Use GPU for displaying instead of CPU. Improves compatibility with screen capture software. May slightly decrease performance.
 STR_5859    :{SMALLFONT}{BLACK}Enables frame tweening for visually{NEWLINE}smoother gameplay. When disabled,{NEWLINE}the game will run at 40 FPS.
 
+#Additional Strings
```

Comment:
You can remove 'Additinonal string' and leave a space there instead, have no idea why there is a 'tooltip' above but I guess we will let that be :v